src/ExpenseTracker/Handlers/Controller.cs

```diff
@@ -9,49 +9,41 @@ namespace ExpenseTracker.Handlers;
 /// </summary>
 public class Controller : IController
 {
-    /// <summary>
-    /// Initializes a new instance of the <see cref="Controller"/> class.
-    /// Sets the user account to access user transactions details.
-    /// </summary>
-    /// <param name="userAccount">Users account.</param>
-    /// <param name="consoleUI">User Interface.</param>
-    public Controller(IAccount userAccount, IUserInterface consoleUI)
-    {
-        this.UserAccount = userAccount;
-        this.UserInterface = consoleUI;
-    }
-
     /// <summary>
     /// Sets the user account to implement user actions.
     /// </summary>
-    /// <value>
-    /// The user account to implement user actions.
-    /// </value>
-    public IAccount UserAccount { private get; set; }
+    private readonly IAccount _userAccount;
 
     /// <summary>
     /// Sets UI object for this controller.
     /// </summary>
-    /// <value>
-    /// User interface object.
-    /// </value>
-    public IUserInterface UserInterface { private get; set; }
+    private readonly IUserInterface _userInterface;
 
     /// <summary>
-    /// Handles menu of expense tracker
+    /// Initializes a new instance of the <see cref="Controller"/> class.
+    /// Sets the user account to access user transactions details.
     /// </summary>
+    /// <param name="userAccount">Users account.</param>
+    /// <param name="consoleUI">User Interface.</param>
+    public Controller(IAccount userAccount, IUserInterface consoleUI)
+    {
+        this._userAccount = userAccount;
+        this._userInterface = consoleUI;
+    }
+
+    /// <inheritdoc/>
     /// <remarks>
     /// Lists all available action options user and prompt the user to select action by index,
     /// calls the respective action handling methods.
     /// </remarks>
     public void HandleMenu()
     {
-        this.UserInterface.MoveToAction(string.Format(Headings.Menu));
+        this._userInterface.MoveToAction(string.Format(Headings.Menu));
         do
         {
-            this.UserInterface.ShowInfoMessage(this.AccountStatsFormat(this.UserAccount.TotalIncome, this.UserAccount.TotalIncome, this.UserAccount.TotalExpense));
-            this.UserInterface.ShowInfoMessage(PromptMessages.MenuPrompt);
-            string? userChoice = this.UserInterface.PromptAndGetInput(PromptMessages.EnterChoice);
+            this._userInterface.ShowInfoMessage(this.AccountStatsFormat(this._userAccount.TotalIncome, this._userAccount.TotalIncome, this._userAccount.TotalExpense));
```

Comment:
```suggestion
            this._userInterface.ShowInfoMessage(this.AccountStatsFormat(
    this._userAccount.TotalIncome,
    this._userAccount.TotalIncome,
    this._userAccount.TotalExpense));
```

Consider splitting the line when it goes off the limit for better readability